CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to the Lua Users Foundation
+
+This document serves as contribution guidelines and the points listed should be
+followed for contributions to the foundation.
+
+Contributions are expected to follow the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Commit Guidelines
+
+As is somewhat standard practice, the following should be observed:
+
+- The usage of Markdown-formatted lists is allowed
+- The header line should be no more than 60 characters
+- Commit headers should start with an imperative verb, to act as a description
+of what applying the commit would do: "If applied, this commit will..."
+- Paragraphs should be broken with an empty line between
+- Further lines after the header should be no more than 72 characters
+- Further lines, if required, should contain detailed information on:
+ 1. Implications of the changes in the commit
+ 2. Reasons the changes are required
+
+Commits relating to one file should follow these guidelines:
+
+- The header line should contain the filename and a short description of what
```

Comment:
Still not sure if enforcing the filename is ideal? I'll let others comment on this

Reply:
It is hard to prepare release notes without this requirement. OTOH, we're not releasing software from this repo...